src/twisted/spread/test/test_pb.py

```diff
@@ -170,6 +169,52 @@ def disconnectClientFactory():
 
 
 
+@implementer(interfaces.IConnector)
+class ReconnectingConnector(object):
+    """
+    An L{IConnector} implementation that calls the specified callback
+    when its C{connect} method is called
+
+    @param reconnectCallback: the callback to call when C{connect} is
+        called
+    @type reconnectCallback: A callable
+
+    @param *args: Position arguments to pass C{reconnectCallback}
```

Comment:
As above, including `*args` is invalid epytext and this should be just `@param args`.

Why isn't pydoctor catching this, I wonder?  Hmm.